browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2052,9 +2052,15 @@ Are you sure you want to do this?
     <message name="IDS_BRAVE_WALLET_NFT_ERC_721" desc="Type of NFT token standard. It does not need to be translated.">
       ERC 721
     </message>
+    <message name="IDS_BRAVE_WALLET_NFT_SOL_SPL" desc="Type of NFT token standard adopted by Solana. It does not need to be translated.">
```

Comment:
add `translateable="false"` to avoid translations for `SPL`

Reply:
I'll add `translateable="false"` to `ERC 721` as well.